src/devices/acurite.c

```diff
@@ -948,7 +1117,7 @@ static int acurite_00275rm_decode(r_device *decoder, bitbuffer_t *bitbuffer)
                 /* clang-format off */
                 data = data_append(data,
                         _X("temperature_1_C", "ptemperature_C"),  "Celsius",      DATA_FORMAT,    "%.1f C",  DATA_DOUBLE, ptempc,
-                        _X("humidity_1", "phumidity"),       "Humidity",     DATA_INT,       phumidity,
+                        _X("humidity_1", "phumidity"),       "Humidity",     DATA_FORMAT, "%u %%", DATA_INT,       phumidity,
```

Comment:
See comment above, would prefer not to have the Atlas PR change the output format for other devices